### DISCUSSION TITLE
Canal 11 Machagai

### DISCUSSION THREAD

**Discussion by Hannaplus4** [iptv-org/community#360] [Show and tell]
### Channel ID

Canal11Machagai.ar

### Stream URL

http://videostream.shockmedia.com.ar:1935/cristalfm/cristalfm/playlist.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Thank you for the stream; it will be included in the next update.

**Comment by freearhey** [comment]
http://videostream.shockmedia.com.ar:1935/cristalfm/cristalfm/playlist.m3u8 (offline)
<img width="538" alt="image" src="https://user-images.githubusercontent.com/7253922/232237016-925a2718-7c77-41b3-b963-0842b2bc15b3.png">
